src/main/js/main_html5.js

```diff
@@ -39,6 +39,14 @@ require("../../../html5-common/js/utils/environment.js");
           list.push(OO.VIDEO.ENCODING.MP4);
         }
 
+        if (!!videoElement.canPlayType("audio/ogg")) {
```

Comment:
I'm just double checking, but you are doing the !! because 'probably' and 'maybe' are okay but empty string ('') isn't.

Reply:
Nevermind, I see that this is the pattern for all of them.